jena-fuseki2/jena-fuseki-webapp/pom.xml

```diff
@@ -57,6 +57,12 @@
       <version>4.5.0-SNAPSHOT</version>
     </dependency>
     
+    <dependency>
+      <groupId>org.apache.jena</groupId>
+      <artifactId>jena-geosparql</artifactId>
+      <version>4.5.0-SNAPSHOT</version>
+    </dependency>
```

Comment:
Ops, I will remove this change. Only added it to test the GeoSparql pull request :+1:

Reply:
Shall I merge then remove this? I'm clearing up prior to 4.5 so in the area.

Reply:
Reverted the change :+1: But thanks for offering to handle this @afs :bow: